### ISSUE TITLE
Won't install against node v14.1

### ISSUE THREAD

**Issue by jorisw** [trevnorris/node-ofe#15]
```
% npm install ofe

> ofe@0.5.1 install /Users/jorisw/Projects/xxx/xxx/node_modules/ofe
> node-gyp rebuild

  CXX(target) Release/obj.target/ofe/ofe.o
In file included from ../ofe.cc:1:
In file included from ../../nan/nan.h:2884:
../../nan/nan_typedarray_contents.h:34:43: warning: 'GetContents' is deprecated: Use GetBackingStore. See http://crbug.com/v8/9908.
      [-Wdeprecated-declarations]
      data   = static_cast<char*>(buffer->GetContents().Data()) + byte_offset;
                                          ^
/Users/jorisw/Library/Caches/node-gyp/14.1.0/include/node/v8.h:5202:3: note: 'GetContents' has been explicitly marked deprecated here
  V8_DEPRECATE_SOON("Use GetBackingStore. See http://crbug.com/v8/9908.")
  ^
/Users/jorisw/Library/Caches/node-gyp/14.1.0/include/node/v8config.h:402:39: note: expanded from macro 'V8_DEPRECATE_SOON'
# define V8_DEPRECATE_SOON(message) [[deprecated(message)]]
                                      ^
../ofe.cc:70:7: error: no type named 'SetFatalErrorHandler' in 'v8::V8'
  V8::SetFatalErrorHandler(OnFatalError);
  ~~~~^
../ofe.cc:74:11: error: no template named 'Handle'
void Init(Handle<Object> target) {
          ^
../ofe.cc:75:7: error: no type named 'SetFatalErrorHandler' in 'v8::V8'
  V8::SetFatalErrorHandler(OnFatalError);
  ~~~~^
../ofe.cc:77:55: error: too few arguments to function call, single argument 'context' was not specified
      Nan::New<FunctionTemplate>(Method)->GetFunction());
      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ ^
/Users/jorisw/Library/Caches/node-gyp/14.1.0/include/node/v8.h:6404:3: note: 'GetFunction' declared here
  V8_WARN_UNUSED_RESULT MaybeLocal<Function> GetFunction(
  ^
/Users/jorisw/Library/Caches/node-gyp/14.1.0/include/node/v8config.h:422:31: note: expanded from macro 'V8_WARN_UNUSED_RESULT'
#define V8_WARN_UNUSED_RESULT __attribute__((warn_unused_result))
                              ^
1 warning and 4 errors generated.
make: *** [Release/obj.target/ofe/ofe.o] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:194:23)
gyp ERR! stack     at ChildProcess.emit (events.js:315:20)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:276:12)
gyp ERR! System Darwin 19.5.0
gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/jorisw/Projects/xxx/xxx/node_modules/ofe
gyp ERR! node -v v14.1.0
gyp ERR! node-gyp -v v5.1.0
gyp ERR! not ok 
npm WARN zenbot4@4.1.0 No license field.
```

macOS Catalina.

**Comment by jorisw** [comment]
Same problem with node/12.10.0 actually.

**Comment by jorisw** [comment]
Nevermind, another similar package won't install either. Think my Xcode is hosed.

**Comment by jorisw** [comment]
`SetFatalErrorHandler` seems to target a very old version of Node, last present in Node 7.10. As such, this package is not very useful until it is updated to target present day Node APIs.

**Comment by trevnorris** [comment]
Didn't realize people still use this. I updated it to work on v10.x+ and published it as version 0.6.0 to npm.
